### PR TITLE
Kanto Mon Names

### PR DESCRIPTION
**Pull request by jd-0**
Relabeling Kanto mons to have their canon English names, being consistent with pokegold/red/etc
addressing issue "Japanese vs. English label names [#77](https://github.com/pret/pokegold-spaceworld/issues/77)"

TODO:
Rename dirs/files, examples:
gfx/pokemon/fushigidane/front.pic
gfx/pokemon/fushigidane/back.pic
gfx/intro/lizardon_1.2bpp

Fix this (??):
dex_order_alpha

Align comments (Or remove them??), ex:
menu_icons
pokemon_constants

### CODE REVIEW

**Comment by Rangi42** [comment]
Thanks for getting this started!

Constant value comments should stay, but they don't need the now-redundant English names. So `const MON_DROWZEE ; 30 DROWZEE` could just be `const MON_DROWZEE ; 30`, and so on. They should still be aligned if they already were.

File: constants/pokemon_constants.asm [hunk 1]
```diff
@@ -2,193 +2,193 @@
...
+	const MON_CUBONE    ; 11 CUBONE
+	const MON_RHYHORN      ; 12 RHYHORN
+	const MON_LAPRAS     ; 13 LAPRAS
+	const MON_ARCANINE      ; 14 ARCANINE
 	const MON_MEW         ; 15 MEW
 	const MON_GYARADOS    ; 16 GYARADOS
 	const MON_SHELLDER    ; 17 SHELLDER
-	const MON_MENOKURAGE  ; 18 TENTACOOL
+	const MON_TENTACOOL  ; 18 TENTACOOL
 	const MON_GHOS        ; 19 GASTLY
```
Comment (Rangi42) [inline constants/pokemon_constants.asm:29]: Should be `MON_GASTLY`.

File: constants/pokedex_constants.asm [hunk 2]
```diff
@@ -1,157 +1,157 @@
...
+	const DEX_FARFETCHD
+	const DEX_DODUO
+	const DEX_DODRIO
+	const DEX_SEEL
+	const DEX_DEWGONG
+	const DEX_GRIMER
+	const DEX_MUK
+	const DEX_SHELLDER
+	const DEX_CLOYSTER
+	const DEX_GHOS
```
Comment (Rangi42) [inline constants/pokedex_constants.asm:95]: Should be `DEX_GASTLY`.

File: constants/pokemon_constants.asm (hunk 1 above)
Comment (jd-0) [reply constants/pokemon_constants.asm]: ah whoops!

**Comment by jd-0** [comment]
Anything else I'm missing? Do I need to re-order dex_order_alpha at all? Files and folders need renaming too still, yes?

**Comment by Rangi42** [comment]
Yes, the TODO items you listed should be added, including renaming the `DEX_*` constants just like all other mentions of the 001-151 Japanese names.

**Comment by jd-0** [comment]
Cool, I think this is ready now!

**Pull request merged**